.github/workflows/deploy-doc.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Doc
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            # ref: "release/0.1.0"
+            submodules: recursive
+
+      - name: Install TorchArrow
+        run: |
+          pip install --pre torcharrow -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
Does the schedule make sure we can get the nightly build on the same day? ^_^

Reply:
I think there's no guarantee --- depending on which one runs first. I'm actually thinking to change this one to manual run only, since it writes to the "release" folder and shouldn't be needed to run nightly. But in this case, we'll need to be careful which version of torcharrow build it installs. Here I'm using the nightly build because that's the one available, but ideally I should use a RC build, which is currently not uploaded anywhere.

On the other hand, we can probably leave a build and deploy doc section to one of the nightly builds, and make sure it uploads the doc to the "main" folder (like [torchdata](https://pytorch.org/data/versions.html)).

But yeah this still needs some tweaks.